src/ontology/edits/oeo-physical.omn

```diff
@@ -1846,14 +1898,14 @@ Class: PlantOperation
         CommonParameters
     
     
-Class: Pollutant
+Class: Pollution
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A pollutant is a substance or energy introduced into the environment that has undesired effects, or adversely affects the usefulness of a resource. A pollutant may cause long- or short-term damage by changing the growth rate of plant or animal species, or by interfering with human amenities, comfort, health, or property values."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Pollutant&oldid=906519011"@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Pollution is an Emission with a negative effect on the environment or organisms."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue #56, pull request #234"
     
     SubClassOf: 
-        <http://purl.obolibrary.org/obo/BFO_0000016>
+        Emission
```

Comment:
I don't think, a subclass relationship is correct here. The emission of a pollutant is the cause for a pollution.

Example: If you spill mud into a pool of clean water the water gets dirty. The mud in the water is the _pollution_ but _emission_ is the process of spilling the mud into the water.

Reply:
The def of Pollution: "Pollution is an Emission with a negative effect on the environment or organisms." refers to the process of polluting. So do we need both the process and the object or is one enough?

Reply:
I think pollution will cause some more discussion, maybe we leave it as it is for this request and discuss it more thoroughly in #263?

Reply:
You might be right, probably I misunderstand. What is then the difference between an emission and a pollution?

Reply:
pollution has a negative impact, emission just an impact

Reply:
Ah, sorry... I read over the little word _negative_. So it is fine to leave it as it is.